packages/create-gatsby/src/questions.json

```diff
@@ -0,0 +1,55 @@
+[
+  {
+    "type": "input",
+    "name": "project",
+    "message": "What would you like to name the folder where you site will be created?",
+    "initial": "my-gatsby-site"
+  },
+  {
+    "type": "select",
+    "name": "cms",
+    "message": "Will you be using a CMS?",
+    "hint": "Use arrow keys to move, and enter to select",
+    "choices": [
+      { "message": "No (or I'll add it later)", "value": "" },
+      { "message": "WordPress", "value": "wordpress" },
+      { "message": "Contentful", "value": "contentful" },
+      { "message": "DatoCMS", "value": "datocms" },
+      { "message": "Sanity", "value": "sanity" },
+      { "message": "AgilityCMS", "value": "agility" }
+    ]
+  },
+  {
+    "type": "select",
+    "name": "styling",
+    "message": "Would you like to install a styling system?",
+    "hint": "Use arrow keys to move, and enter to select",
+    "choices": [
+      { "message": "No (or I'll add it later)", "value": "" },
+      { "message": "CSS Modules/PostCSS", "value": "postcss" },
+      { "message": "Styled Components", "value": "styled-components" },
+      { "message": "Emotion", "value": "emotion" },
+      { "message": "Sass", "value": "sass" },
+      { "message": "Theme-UI", "value": "themeui" }
+    ]
+  },
+  {
+    "type": "multiselect",
+    "name": "features",
+    "message": "Would you like to install additional features with other plugins?",
+    "hint": "Use arrow keys to move, spacebar to select, and enter to confirm your choices",
+    "choices": [
```

Comment:
I tried referring to the top 20 plugins we added schemas for to add some ideas here (because that would be required for us to install these).

Reply:
Tiny nitpicky clarification: the schema isn't required to _install_ them, but for the best experience we'd want to only recommend plugins with schemas as then we can also prompt the users to configure them correctly. 👍

Reply:
I think we're going to require it for "plugin add"